server/committers.go

```diff
@@ -0,0 +1,35 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"sourcegraph.com/sourcegraph/go-vcs/vcs"
+)
+
+func (h *Handler) serveRepoCommitters(w http.ResponseWriter, r *http.Request) error {
+	repo, _, done, err := h.getRepo(r)
+	if err != nil {
+		return err
+	}
+	defer done()
+
+	// TODO: implement fetching CommittersOptions from the URL query string.
```

Comment:
This was trivial to add, so I've added it in, and updated the vcsclient code to send CommittersOptions via the query string.